app/controllers/api/v1/registrations_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Api
   module V1
     class RegistrationsController < DeviseTokenAuth::RegistrationsController
+      skip_before_action :authenticate_user!, raise: false
```

Comment:
I believe this is not necessary here as it is the registrations controller, which should not authenticate the user.